chromium_src/third_party/blink/renderer/modules/clipboard/clipboard_promise.cc

```diff
@@ -0,0 +1,13 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "third_party/blink/renderer/modules/clipboard/clipboard_promise.h"
+
+#define BRAVE_CLIPBOARD_PROMISE_REQUEST_PERMISSION \
+  false);                                          \
+ if(
```

Comment:
lgtm, but imo quite unclear change for a pretty critical issue.

maybe just add our own block to prevent any clipboard interaction without user gesture? (and add test?)
```
  if (!has_transient_user_activation) {
    script_promise_resolver_->Reject(MakeGarbageCollected<DOMException>(
        DOMExceptionCode::kSecurityError,
        "Must be handling a user gesture to use clipboard"));
    return;
  }
```

Reply:
@goodov agree that we may want to add our check, but I'd prefer to limit this fix to just reverting the Chromium's temporary(?) relaxation of the gesture requirement. @ShivanKaul do we need a follow up issue to consider adding our own restriction here as @goodov suggesting?